tasks/configure_alternatives.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Configure all alternatives for OpenJDK 11
+  community.general.alternatives:
```

Comment:
Can you please include a comment here that points folks to where the list of alternatives came from?

Reply:
Please see commit fafd4b5.

Reply:
Perfect, thanks!